app/src/main/java/com/asfoundation/wallet/interact/FindDefaultWalletInteract.kt

```diff
@@ -0,0 +1,25 @@
+package com.asfoundation.wallet.interact
+
+import com.asfoundation.wallet.entity.Wallet
+import com.asfoundation.wallet.repository.WalletRepositoryType
+import io.reactivex.Single
+import io.reactivex.schedulers.Schedulers
+
+class FindDefaultWalletInteract(private val walletRepository: WalletRepositoryType) {
+  fun find(): Single<Wallet> {
+    return walletRepository.defaultWallet.subscribeOn(Schedulers.io())
```

Comment:
According to our architecture guideline Schedulers should be passed in the constructor